### PR TITLE
Death flags

### PR DESCRIPTION
**Pull request by runtd**
## Changelog:
- Added coat of arms for death faiths.

### CODE REVIEW

File: common/coat_of_arms/template_lists/colored_emblem_lists.txt [hunk 1]
```diff
@@ -781,6 +781,24 @@ colored_emblem_texture_lists = {
...
+			}
+			100 	= "ce_abomination.dds"
+			100 	= "ce_dk_helmet.dds"
+			100 	= "ce_necropolis.dds"
+			100 	= "ce_ram_skull.dds"
+			100 	= "ce_scream.dds"
+			100 	= "ce_skull1.dds"
+			100 	= "ce_skull1_fire.dds"
+			100 	= "ce_skull2.dds"
+			100 	= "ce_skull3.dds"
```
Comment (zumbak04) [inline common/coat_of_arms/template_lists/colored_emblem_lists.txt:798]: Should be bigger and lowered.
<details>
<summary>Click to expand</summary>

image
image

</details>

File: common/coat_of_arms/template_lists/colored_emblem_lists.txt [hunk 2]
```diff
@@ -781,6 +781,24 @@ colored_emblem_texture_lists = {
...
+			100 	= "ce_skull3_jaw.dds"
+			100 	= "ce_spider.dds"
```
Comment (zumbak04) [inline common/coat_of_arms/template_lists/colored_emblem_lists.txt:800]: Should be lowered.
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]
Just want to say that this CoA looks ideal. Fits well.
<details>
<summary>Click to expand</summary>

image

</details>

File: common/coat_of_arms/template_lists/colored_emblem_lists.txt (hunk 1 above)
Comment (zumbak04) [reply common/coat_of_arms/template_lists/colored_emblem_lists.txt:798]: Still too small.
<details>
<summary>Click to expand</summary>

image

</details>

File: common/coat_of_arms/template_lists/colored_emblem_lists.txt [hunk 3]
```diff
@@ -781,6 +781,24 @@ colored_emblem_texture_lists = {
 			}
 			100 	= "ce_kirintor_eye.dds"
 		}
+		
+		### Death
+		special_selection = {
+			trigger = {
+				death_coa_trigger = yes
+			}
+			100 	= "ce_abomination.dds"
```
[the remark this answers is not in the document]
Comment (zumbak04) [reply common/coat_of_arms/template_lists/colored_emblem_lists.txt:790]: I would increase the contrast. It's very hard to notice these features.
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

**Review by zumbak04 — APPROVED** [review]
LGTM! Good job!

**Review by TheDrPibb1999 — APPROVED** [review]
Personally enjoy all, LGTM

**Pull request merged**